sites/healthcarepackaging/server/graphql/fragments/content-company.js

```diff
@@ -0,0 +1,94 @@
+const gql = require('graphql-tag');
+
+module.exports = gql`
+
+fragment WebsiteContentCompanyFragment on Content {
```

Comment:
Since this fragment is _so_ large, can we move it to common: `/common/graphql/fragments/content-company.js`?
You'll need to up date the imports, obviously